packages/contracts/contracts/optimistic-ethereum/libraries/utils/Lib_StringUtils.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >0.5.0 <0.8.0;
+
```

Comment:
This file is helpful for rendering dynamic revert messages

Reply:
Happy to remove this file since its only used in the dynamic revert message I added for debugging purposes